packages/bank_gateway/data/schema.js

```diff
@@ -30,6 +31,21 @@ const typeDefs = `
         is_active: Boolean,
         _id: String
     }
+
+    type MoneyTransfer {
+        t_id :Number,
+        rec_accno:Number,    
+        rec_Fname :String,   
+        rec_Lname:String,    
+        bank_name:String,    
+        ifsc_code:String,    
+        mob_no:Number,       
+        amt:Number,          
+        remark:String,       
+        acc_no:Number,        
+        trans_type: String, 
+        customer_id:Number   
+    }
```

Comment:
There is no Number datatype in graphql. This will give error. Apply following types
```suggestion
    type MoneyTransfer {
        t_id: ID,
        rec_accno: ID,    
        rec_Fname: String,   
        rec_Lname: String,    
        bank_name: String,    
        ifsc_code: String,    
        mob_no: Int,       
        amt: Float,          
        remark: String,       
        acc_no: Int,        
        trans_type: String, 
        customer_id: Int
    }
```